content/source/docs/enterprise/workspaces/repo-structure.html.md

```diff
@@ -7,13 +7,19 @@ sidebar_current: "docs-enterprise2-workspaces-repo-structure"
 # Repository Structure
 
 Terraform Enterprise integrates with version control repositories to obtain
-configurations and trigger Terraform runs. Structuring the repo properly is
+configurations and trigger Terraform runs. Structuring these repos properly is
 important because it determines which files Terraform has access to when
 Terraform is executed within Terraform Enterprise, and when Terraform plans will run.
 
+## Manageable Repos
+
+As a best practice for repository structure, each repository containing Terraform code should be a [manageable chunk of infrastructure](/docs/enterprise/guides/recommended-practices/part1.html#the-recommended-terraform-workspace-structure), such as an application, service, or specific type of infrastructure (like common networking infrastructure).
+
+When repositories are interrelated, using [remote state](/docs/enterprise/guides/recommended-practices/part3.3.html#3-design-your-organization-s-workspace-structure) to transfer information between workspaces is recommended. Using small configurations connected by remote state means that each workspace will only run plans on relevant updates. It prevents the situation where a single repository is connected to a large number of workspaces and many plans run on every update, only a few of which have potential changes.
```

Comment:
```suggestion
When repositories are interrelated, we recommend using [remote state](/docs/enterprise/guides/recommended-practices/part3.3.html#3-design-your-organization-s-workspace-structure) to transfer information between workspaces. Small configurations connected by remote state are more efficient for collaboration than monolithic repos, because they let you update infrastructure without running unnecessary plans in unrelated workspaces.
```

...or something like that; I just wanted to get something in there about _why_ you would want to help us not waste SaaS resources. :) 

Oh hey, they made suggestions match the soft wrap behavior of the code they're affecting! That was another thing I sent in on the feedback form.

Reply:
Oh, bah, I guess it only soft wraps if you're viewing it in the "files changed" tab, not the summary.